app/helpers/talks_helper.rb

```diff
@@ -0,0 +1,25 @@
+# Talks Helper
+module TalksHelper
+  def talk_positions_for_select(talk)
+    [[t('basics.at_the_beginning'), 0]] + talk.lecture.select_talks -
+      [[talk.to_label, talk.position]]
+  end
+
+  def talk_card_color(talk, user)
+    return 'bg-mdb-color-lighten-2' unless user.in?(talk.speakers)
+    'bg-info'
+  end
+
+  def speaker_list(talk)
+    return t('basics.tba') unless talk.speakers.present?
+    talk.speakers.map(&:tutorial_name).join(', ')
+  end
```

Comment:
I think this function is at least not used in two situations (app/views/media/_medium.html.erb and app/views/talks/_card.html.erb)

Reply:
Rewritten so that it is in use now.